meta-lmp-support/recipes-bsp/u-boot/u-boot-fio/uz/0001-Read-MAC-from-EEPROM-on-Xilinx.patch

```diff
@@ -0,0 +1,42 @@
+From 639eb84dad63d917b325b25fabefebb4f610e3ec Mon Sep 17 00:00:00 2001
+From: Ari Parkkila <ari.parkkila@pelion.com>
+Date: Wed, 24 Feb 2021 15:50:45 +0200
+Subject: [PATCH] Read MAC from EEPROM on Xilinx
+
+
+diff --git a/meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/device-tree/device-tree/uz3eg-iocc/system-board.dtsi b/meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/device-tree/device-tree/uz3eg-iocc/system-board.dtsi
+index 99eced2..353a111 100644
+--- a/meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/device-tree/device-tree/uz3eg-iocc/system-board.dtsi
++++ b/meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/device-tree/device-tree/uz3eg-iocc/system-board.dtsi
+@@ -30,6 +30,11 @@
+ 			linux,default-trigger = "heartbeat";
+ 		};
+ 	};
++
++        chosen {
++                xlnx,eeprom = &eeprom; /* EMAC on EEPROM */
++        };
++
+ };
+ 
+ /* Ethernet0 with hard-coded MAC */
+@@ -37,7 +42,6 @@
+ 	status = "okay";
+ 	phy-handle = <&phy0>;
+ 	phy-mode = "rgmii-id";
+-	local-mac-address = [00 0a 35 00 02 90];
```

Comment:
WHat's the backup if there's no mac written to EEPROM? Or is such a situation even possible?

Reply:
Ah I read the PR description after writing this comment